packages/page-header/PageHeader.js

```diff
@@ -30,7 +30,7 @@ const PageHeader = ({
       <div className="d-flex align-items-start">
         <Breadcrumbs crumbs={crumbs} active={appName || children} /> {component}
       </div>
-      <h1 className="h4 page-header page-header-brand" {...props}>
+      <h4 className="page-header page-header-brand" {...props}>
```

Comment:
This is just as much as an opinion as the `h1` as setting `h4` assumes that the page already defines `h1`, `h2`, and `h3` before-hand since you shouldn't skip a heading level.

Maybe this tag should be configuration (similar to the `tag` prop in reactstrap). Since this is supposed to be _the_ page header, and typically (branding) it is the name of the app/page and the most top level header, it could default to `h1` but be overridable for less standard uses elsewhere within the page.